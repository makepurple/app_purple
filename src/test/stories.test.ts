/* eslint-disable @typescript-eslint/no-unsafe-call */
import initStoryshots from "@storybook/addon-storyshots";

initStoryshots();
