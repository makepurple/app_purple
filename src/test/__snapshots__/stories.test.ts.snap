// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots atoms/Button Standard 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-sizing: border-box;
  padding: 14px 16px;
  border-radius: 0.375rem;
  border: none;
  -webkit-transition: opacity 0.15s cubic-bezier(0.4,0,0.2,1);
  transition: opacity 0.15s cubic-bezier(0.4,0,0.2,1);
  font-size: 1.125rem;
  line-height: 1.125em;
  font-weight: 500;
  background-color: #8E8CFF;
  color: #ffffff;
  cursor: pointer;
}

.c0:hover {
  opacity: 0.85;
}

<button
  className="c0"
>
  Click me!
</button>
`;

exports[`Storyshots atoms/Paper Standard 1`] = `
.c0 {
  border-radius: 0.5rem;
  box-shadow: 0 10px 15px -3px rgba(0,0,0,0.1),0 4px 6px -2px rgba(0,0,0,0.05);
  background-color: #ffffff;
}

<div
  className="c0"
>
  Quisquam recusandae alias consequuntur corporis repellat ratione ut sunt qui. Iure ut libero qui. Ut nulla quam ipsam nobis cupiditate sed dignissimos debitis incidunt. Sed libero repudiandae.
 
Natus et eos itaque velit omnis. Porro ut et ipsam explicabo eligendi occaecati debitis et. Eum dicta eum eaque enim ipsum inventore debitis libero aspernatur. Quam tempore a velit provident velit eligendi.
 
Qui nobis repellendus fugiat velit. Aperiam placeat fuga. Distinctio placeat ullam minima ducimus temporibus modi aut architecto ducimus. Voluptates explicabo exercitationem ut quis sed.
 
Sit vitae soluta sint non voluptates ut. Quos qui illo error sunt. Ratione a officia quaerat cumque incidunt aut.
 
Hic eligendi quos esse ut ab. Sed quae nam eos sint autem rerum doloremque officia. Ut ut eos ducimus eos saepe consequatur vel in ut. Et molestiae ea ut. Placeat quidem est consequuntur aut est fuga.
 
Ex ut voluptas enim ex eveniet facere molestiae aut. Aut nam et dolorum architecto fugit repellendus hic veritatis qui. Culpa earum voluptate vel labore. Ut est sunt corporis alias est et sequi voluptas. Non aut rerum impedit ex rem voluptates.
 
Totam perferendis ut non fugiat eligendi omnis. Quasi ipsum rem eos quod recusandae consectetur optio laudantium et. Possimus molestias facere quia. Ab quo occaecati quia ipsum sit qui accusantium odit doloremque. Dolore aut eos quis ut incidunt omnis. Voluptas minima quia ut minima sunt qui similique ut quam.
 
Sit vel et nostrum ut. Aut laborum iure molestiae et facere placeat molestiae. Molestiae dolores commodi inventore sequi quia itaque. Sint odit dolor. Et quo aperiam natus.
 
Magni quasi exercitationem. Ea similique ad sed architecto magnam voluptatem maxime neque adipisci. Culpa quis ut ab nulla quod labore ullam quos sed. Voluptatibus eum saepe nesciunt omnis sit et. Eaque corrupti vel.
 
Similique blanditiis voluptatem provident doloribus aut culpa ratione vel. Omnis numquam maxime sint libero laudantium voluptatem. Et voluptas quis non. Architecto non explicabo facere neque et ex quaerat. Aut qui adipisci.
</div>
`;
